ppa/archive/admin.py

```diff
@@ -59,6 +60,30 @@ def source_link(self, obj):
     source_link.admin_order_field = 'source_id'
     source_link.allow_tags = True
 
+    def save_model(self, request, obj, form, change):
+        '''Note any fields in the protected list that have been changed
+        and preserve in database.'''
+        # new object, created from scratch, nothing to track and preserve
+        # NOTE: Is this logic correct? Or should all the fields be preserved?
```

Comment:
This logic seems right to me - we don't expect (or support) new records for HathiTrust items to be created from django admin.

However, it does occur to me that the protected field flags are hathi specific - I think this logic should only trigger when source=hathi.

Reply:
I will add a flag for this re: hathi